src/unroll_property.py

```diff
@@ -0,0 +1,26 @@
+"""Helper functions to unroll gremlin list property"""
+
+from src.sanitizer import unsanitize
+
+def value(keys):
+    """Nested dictionary keys who's value to be unrolled"""
+    def _get(obj) -> str:
+        try:
+            for key in keys.split('.'):
+                obj = obj[key]
+            # old versions of gremlin don't have proper way to
+            # unroll a list to value who's size is 1.
+            return obj[0]
+        except (IndexError, TypeError, KeyError):
+            pass
```

Comment:
incase of exception we are simply passing, in this case we wont know if it's breaking. You can log the error at least.  In case of pass you can return None here only.

Reply:
Attribute assignment happens during development stage and it is not dynamic.
So it is expected to pass and return None. Actually this implementation is inspired from[ flask-restplus](https://github.com/noirbizarre/flask-restplus/blob/master/flask_restplus/fields.py#L63)